README.md

```diff
@@ -1,5 +1,10 @@
 # Task_tests
```

Comment:
Should contain steps for project setup too.

Reply:
in separate text file ?

Reply:
README.md is a good place for this